lib/operations/arithmetic/divide.js

```diff
@@ -0,0 +1,6 @@
+function divide(a, b) {
+  return a / b;
+}
+
+divide.code = '/';
```

Comment:
Same here `code` as described in https://github.com/axa-ch/json-logic-js/pull/1/files#r259730014

`/` is an existing operator